pkg/account/storage/v2/account.go

```diff
@@ -289,3 +297,12 @@ func (s *accountStorage) ListAccountsV2(
 	}
 	return accounts, nextOffset, totalCount, nil
 }
+
+func (s *accountStorage) IsNewUser(ctx context.Context, email string) (bool, error) {
```

Comment:
Instead of adding a new interface, please use `GetAccount` and check the `NotFound` error to maintain consistency with the other places.

Reply:
```go
GetAccountV2(ctx context.Context, email, organizationID string)
```
GetAccountV2 need `organizationId` param, we don't need it when check if a use if new registration